components/org.wso2.carbon.micro.integrator.server/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Old license headers

Reply:
I suppose having 2016 is ok given that we are moving module instead of creating new ones.